rules/windows/execution_suspicious_short_program_name.toml

```diff
@@ -0,0 +1,27 @@
+[metadata]
+creation_date = "2020/11/15"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/15"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies process execution with a single character process name. This is often done by adversaries while staging or
+executing temporary utilities.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Suspicious Execution - Short Program Name"
+risk_score = 47
+rule_id = "17c7f6a5-5bc9-4e1f-92bf-13632d24384d"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Execution"]
+type = "eql"
+
+query = '''
+process where event.type in ("start", "process_started") and length(process.name) > 0 and
+ length(process.name) == 5 and host.os.name == "Windows" and length(process.pe.original_file_name) > 5
```

Comment:
what if there is no original file name? then you won't get any results

Reply:
its optional, will add hint uncomment if too noisy